src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -23,8 +22,10 @@ def get(self, request: Request, organization) -> Response:
         except NoProjects:
             return Response({"count": 0})
 
+        dataset = self.get_dataset(request)
```

Comment:
The other endpoints used in Discover do this conditional check for feature flags https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_events.py#L252-L257 
and I'm not really sure why this one doesn't, is it safe to not have one here? I still need to switch the dataset based on which param is passed, but I guess a user could hard code the param and what ... access data for events they don't have?

Reply:
Hmm good catch, its not here mostly cause we(I) missed it. I think its low risk here so fine to skip :+1: